data/phi0474/phi019/phi0474.phi019.perseus-eng2.xml

```diff
@@ -120,21 +124,28 @@
    <change when="2007-01-04" who="packel">fixed the refsDecls for Cornelius</change>
    <change when="2006-05-05" who="packel">fixed date tags</change>
    <change when="2006-02-10" who="packel">places/dates now actually tagged</change>
-   <change when="2005-11-21" who="lcerrato">changed thought you shall not perceive them to though...</change>
-   <change when="2005-11-21" who="lcerrato">changed: does not the precaution taken of assembling the senate in thus most defensible place thus to this</change>
+   <change when="2005-11-21" who="lcerrato">changed thought you shall not perceive them to
```

Comment:
hey @lcerrato that's so strange, I didn't even touch the change log for this file, just changed the bibliographic information, maybe I accidentally did a pretty print or format and indent, because I didn't manually change any of those item.

Reply:
hi @lcerrato even though there is some weird spacing since everything else is OK, can I merge this pull request? Iwill make sure not to pretty print any other files I promise!

Reply:
Did you change that little typo?

Reply:
Just did, whoops!